dev-docs/ui/technology-overview.adoc

```diff
@@ -0,0 +1,83 @@
+= Technology Overview
```

Comment:
I think "sheer number" is okay, have you ever looked at a React app?  "Now downloading library 234 of 789"....

Reply:
I listed all libraries here in detail, even though some of them are common libraries in Kotlin / Compose projects. I did this only because most of the libraries are for non-kotlin folks new, so having some reference should be helpful for getting started. Maybe in the long run it doesn't have to be that detailed.

Reply:
Yeah, it ended up not being too bad once I got into the actual PR code.  The docs set great context though 👍